python/lsst/pipe/base/_datasetQueryConstraints.py

```diff
@@ -117,7 +117,7 @@ def fromExpression(cls, expression: str) -> DatasetQueryConstraintVariant:
 else:
 
     class MetaMeta(type(Protocol)):
-        def __init__(cls, *args, **kwargs):
+        def __init__(cls, *args, **kwargs):  # noqa: N805
```

Comment:
Interesting that `ruff check` doesn't think this is an error. We don't use flake8 for this package so not sure how you found this but I confirm flake8 is upset.

Reply:
I may have run `flake8` on the code locally out of habit.